From a6e907c2acb3ca370a4d1e6cfe0c4d16ca310421 Mon Sep 17 00:00:00 2001
From: Tony Lin <tony.lin@freescale.com>
Date: Thu, 2 Feb 2012 13:59:05 +0800
Subject: ENGR00173615-1 [mx6q]add mx6q_revision to common header file

the common API is needed by drivers to distinguish TO version

Signed-off-by: Tony Lin <tony.lin@freescale.com>
---
 arch/arm/plat-mxc/include/mach/mxc.h |    7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/arch/arm/plat-mxc/include/mach/mxc.h b/arch/arm/plat-mxc/include/mach/mxc.h
index 628ef14..7ce96e0 100755
--- a/arch/arm/plat-mxc/include/mach/mxc.h
+++ b/arch/arm/plat-mxc/include/mach/mxc.h
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2007, 2011 Freescale Semiconductor, Inc.
+ * Copyright 2004-2007, 2012 Freescale Semiconductor, Inc.
  * Copyright (C) 2008 Juergen Beisert (kernel@pengutronix.de)
  *
  * This program is free software; you can redistribute it and/or
@@ -90,6 +90,11 @@ extern unsigned int system_rev;
 	board_is_rev(IMX_BOARD_REV_2))
 #define board_is_mx6q_sabre_auto() (cpu_is_mx6q() && \
 	board_is_rev(IMX_BOARD_REV_3))
+
+#ifndef __ASSEMBLY__
+extern int mx6q_revision(void);
+#endif
+
 #endif
 
 #ifndef __ASSEMBLY__
-- 
1.7.9.5

